ui/compose.ui

```diff
@@ -39,6 +39,8 @@ services:
     network_mode: "host"
     depends_on:
       - redis
+    env_file:
```

Comment:
I think you also need to fix the [deploy/compose/deplo](https://github.com/instructlab/instructlab-bot/blob/main/deploy/compose/dev-single-worker-with-ui.yaml)